include/networkit/auxiliary/GlobalState.hpp

```diff
@@ -0,0 +1,71 @@
+#ifndef NETWORKIT_AUXILIARY_GLOBAL_STATE_HPP_
+#define NETWORKIT_AUXILIARY_GLOBAL_STATE_HPP_
+
+// networkit-format
+
+#include <atomic>
+#include <csignal>
+#include <cstdint>
+#include <fstream>
+#include <mutex>
+
+#include <networkit/auxiliary/Log.hpp>
+#include <networkit/auxiliary/SignalHandling.hpp>
+
+#include <networkit/networkit_export.h>
+
+namespace Aux {
+
+namespace GlobalState {
```

Comment:
Instead of using `Aux::GlobalState`, I'd prefer `NetworKit::GlobalState`. Further namespaces within `GlobalState` should not be necessary.